Base/usr/share/man/man1/stat.md

```diff
@@ -0,0 +1,36 @@
+## Name
+
+stat
+
+## Synopsis
+
+```**sh**
+$ stat [-L] file
+```
+
+## Description
+
+Display file status.
+
+## Options:
```

Comment:
I realize that the colon is present in a couple of man pages already, but headings should never have one (which the majority of man pages agrees with). Please remove it.